homeassistant/components/alarm_control_panel/__init__.py

```diff
@@ -150,6 +155,19 @@ class AlarmControlPanelEntity(Entity, cached_properties=CACHED_PROPERTIES_WITH_A
     _attr_supported_features: AlarmControlPanelEntityFeature = (
         AlarmControlPanelEntityFeature(0)
     )
+    _alarm_control_panel_option_default_code: str = ""
+
+    @final
+    @callback
+    def add_default_code(self, code: str | None) -> str | None:
+        """Add default arm code."""
```

Comment:
The doc string is lying, maybe
```suggestion
        """Return code to use for a service call.

        If the passed in code is not None, it will be return. Otherwise return the
        default code if set, or None if not set.
        """
```